docker/geoserver/Dockerfile

```
@@ -1,3 +1,3 @@
-FROM docker.terrestris.de/terrestris/geoserver:2.21.0
+FROM docker.terrestris.de/terrestris/geoserver:2.25.1
```

Not essentially needed, but we might want to make use of the official image instead:

```suggestion
FROM docker.osgeo.org/geoserver:2.25.1
```